modules/python/test/test.py

```diff
@@ -23,48 +23,16 @@
 except ImportError:
```

Comment:
use run the test.py by passing it to python, probably we should get back to 644 permissions instead of 755